### ISSUE TITLE
setup.py fails on Windows

### ISSUE THREAD

**Issue by jeandudey** [spesmilo/electrum#1033]
Happens with this command:
```
python setup.py install
```

This is the output:
```
Including all files
running install
running bdist_egg
running egg_info
writing requirements to Electrum.egg-info\requires.txt
writing Electrum.egg-info\PKG-INFO
writing top-level names to Electrum.egg-info\top_level.txt
writing dependency_links to Electrum.egg-info\dependency_links.txt
file gui\qt\icons_rc.py (for module electrum_gui.qt.icons_rc) not found
reading manifest file 'Electrum.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*.py' under directory 'packages'
warning: no files found matching 'cacert.pem' under directory 'packages'
warning: no files found matching '*.mo' under directory 'locale'
writing manifest file 'Electrum.egg-info\SOURCES.txt'
installing library code to build\bdist.win32\egg
running install_lib
running build_py
file gui\qt\icons_rc.py (for module electrum_gui.qt.icons_rc) not found
copying lib\__init__.py -> build\lib\electrum
copying lib\account.py -> build\lib\electrum
copying lib\bitcoin.py -> build\lib\electrum
copying lib\blockchain.py -> build\lib\electrum
copying lib\bmp.py -> build\lib\electrum
copying lib\commands.py -> build\lib\electrum
copying lib\daemon.py -> build\lib\electrum
copying lib\i18n.py -> build\lib\electrum
copying lib\interface.py -> build\lib\electrum
copying lib\mnemonic.py -> build\lib\electrum
copying lib\msqr.py -> build\lib\electrum
copying lib\network.py -> build\lib\electrum
copying lib\network_proxy.py -> build\lib\electrum
copying lib\old_mnemonic.py -> build\lib\electrum
copying lib\paymentrequest.py -> build\lib\electrum
copying lib\paymentrequest_pb2.py -> build\lib\electrum
copying lib\plugins.py -> build\lib\electrum
copying lib\qrscanner.py -> build\lib\electrum
copying lib\simple_config.py -> build\lib\electrum
copying lib\synchronizer.py -> build\lib\electrum
copying lib\transaction.py -> build\lib\electrum
copying lib\util.py -> build\lib\electrum
copying lib\verifier.py -> build\lib\electrum
copying lib\version.py -> build\lib\electrum
copying lib\wallet.py -> build\lib\electrum
copying lib\x509.py -> build\lib\electrum
copying gui\__init__.py -> build\lib\electrum_gui
copying gui\gtk.py -> build\lib\electrum_gui
copying gui\qt\__init__.py -> build\lib\electrum_gui\qt
copying gui\qt\__init__.py -> build\lib\electrum_gui\qt
copying gui\qt\amountedit.py -> build\lib\electrum_gui\qt
copying gui\qt\console.py -> build\lib\electrum_gui\qt
copying gui\qt\history_widget.py -> build\lib\electrum_gui\qt
copying gui\qt\installwizard.py -> build\lib\electrum_gui\qt
copying gui\qt\lite_window.py -> build\lib\electrum_gui\qt
copying gui\qt\main_window.py -> build\lib\electrum_gui\qt
copying gui\qt\network_dialog.py -> build\lib\electrum_gui\qt
copying gui\qt\password_dialog.py -> build\lib\electrum_gui\qt
copying gui\qt\paytoedit.py -> build\lib\electrum_gui\qt
copying gui\qt\qrcodewidget.py -> build\lib\electrum_gui\qt
copying gui\qt\qrtextedit.py -> build\lib\electrum_gui\qt
copying gui\qt\qrwindow.py -> build\lib\electrum_gui\qt
copying gui\qt\receiving_widget.py -> build\lib\electrum_gui\qt
copying gui\qt\seed_dialog.py -> build\lib\electrum_gui\qt
copying gui\qt\transaction_dialog.py -> build\lib\electrum_gui\qt
copying gui\qt\util.py -> build\lib\electrum_gui\qt
copying gui\qt\version_getter.py -> build\lib\electrum_gui\qt
copying gui\stdio.py -> build\lib\electrum_gui
copying gui\text.py -> build\lib\electrum_gui
copying plugins\__init__.py -> build\lib\electrum_plugins
copying plugins\btchipwallet.py -> build\lib\electrum_plugins
copying plugins\coinbase_buyback.py -> build\lib\electrum_plugins
copying plugins\cosigner_pool.py -> build\lib\electrum_plugins
copying plugins\exchange_rate.py -> build\lib\electrum_plugins
copying plugins\greenaddress_instant.py -> build\lib\electrum_plugins
copying plugins\labels.py -> build\lib\electrum_plugins
copying plugins\trezor.py -> build\lib\electrum_plugins
copying plugins\virtualkeyboard.py -> build\lib\electrum_plugins
copying plugins\plot.py -> build\lib\electrum_plugins
file gui\qt\icons_rc.py (for module electrum_gui.qt.icons_rc) not found
installing package data to build\bdist.win32\egg
running install_data
Traceback (most recent call last):
  File "setup.py", line 150, in <module>
    long_description="""Lightweight Bitcoin Wallet"""
  File "C:\Python27\lib\distutils\core.py", line 152, in setup
    dist.run_commands()
  File "C:\Python27\lib\distutils\dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "C:\Python27\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\lib\site-packages\setuptools\command\install.py", line 67, i
n run
    self.do_egg_install()
  File "C:\Python27\lib\site-packages\setuptools\command\install.py", line 109,
in do_egg_install
    self.run_command('bdist_egg')
  File "C:\Python27\lib\distutils\cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "C:\Python27\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\lib\site-packages\setuptools\command\bdist_egg.py", line 181
, in run
    self.do_install_data()
  File "C:\Python27\lib\site-packages\setuptools\command\bdist_egg.py", line 133
, in do_install_data
    self.call_command('install_data', force=0, root=None)
  File "C:\Python27\lib\site-packages\setuptools\command\bdist_egg.py", line 147
, in call_command
    self.run_command(cmdname)
  File "C:\Python27\lib\distutils\cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "C:\Python27\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\lib\distutils\command\install_data.py", line 58, in run
    dir = convert_path(f[0])
  File "C:\Python27\lib\distutils\util.py", line 202, in convert_path
    raise ValueError, "path '%s' cannot end with '/'" % pathname
ValueError: path 'C:\Python27\share\applications/' cannot end with '/'
```

**Comment by ecdsa** [comment]
it should be fixed in b5ac7f2a7c5b674613e5ee9b6164cece1511db07
please confirm

**Comment by jeandudey** [comment]
It works!
